### DISCUSSION TITLE
Are all classes supported? unknown or invalid utility with container, text-overflow, etc.

### DISCUSSION THREAD

**Discussion by nikkwong** [jaredh159/tailwind-react-native-classnames#215] [Q&A]
There are several classes that don't seem to be supported, such as `pointer-events-none`, `container`, `whitespace-nowrap` as they throw the warning `unknown or invalid utility`.

I'm wondering if these are actually not supported and if so if there is a list somewhere of which classes are not included in this library.

**Comment by jaredh159** [comment]
it's important to know that this library _only_ intends (currently) to provide support for tailwindcss utilities that **have a counterpart in RN Native (not web)**.  I'm not sure about all of the ones you mentioned, but I think at least `pointer-events-none` would be RN Web only, maybe all of them. If you can find a supported RN Native style prop mapping to any of those, feel free to open an issue or a PR. But if it's only going to affect RN Web projects, that's currently out of scope for this project.

here's a list of supported utilities:

https://github.com/jaredh159/tailwind-react-native-classnames/blob/master/supported-utilities.md

See here for more context on RN Web:

https://github.com/jaredh159/tailwind-react-native-classnames/discussions/125
